keep-ui/app/settings/users-menu.tsx

```diff
@@ -9,13 +9,18 @@ import { getApiURL } from "utils/apiUrl";
 import { User } from "./models";
 import { User as AuthUser } from "next-auth";
 import { mutate } from "swr";
+import { useFloating } from "@floating-ui/react-dom";
```

Comment:
This package isn't necessary installed (it's a peer dependency for one of the UI libraries). If we'll be using it to position menus, we should install it. The newer versions have an improved API as well.

Reply:
I've copied the code from `alert-menu` so maybe @talboren can give a bit more context.
Anyway I'll add it until we decide to move from it.

Reply:
TBH, I think that's something @asharonbaltazar you added

Reply:
I didn't add it! It was already in the project, but it's not part of the project dependencies...